.github/workflows/deploy.yml

```diff
@@ -0,0 +1,38 @@
+name: Deploy
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  release:
+    name: Deploy
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12.x
+
+      - name: get yarn cache dir
+        id: yarn-cache
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+
+      - uses: actions/cache@v1
+        with:
+          path: ${{ steps.yarn-cache.outputs.dir }}
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+
+      - name: install dependencies
+        run: yarn install
+
+      - name: Deploy
+        env:
+          NOW_AUTH_TOKEN: ${{ secrets.NOW_TOKEN }}
+        run: |
+          now deploy --token=$NOW_AUTH_TOKEN --scope=e-simple-auth && now alias --token=$NOW_AUTH_TOKEN --scope=e-simple-auth
+          cd server && now deploy --token=$NOW_AUTH_TOKEN --scope=e-simple-auth && now alias --token=$NOW_AUTH_TOKEN --scope=e-simple-auth
```

Comment:
do we need both?

Reply:
Yes, one deploys the Ember app, the other one deploys the mock server.

Reply:
it seems to be the exact same call. will they be deployed to different locations? 🤔

Reply:
Yes, one has `"name": "simple-auth-server",` in the `package.json`, the other is `"name": "ember-simple-auth",` - see the `cd server`

Reply:
I see 👍